Cargo.toml

```diff
@@ -49,8 +49,14 @@ bincode1 = { package = "bincode", version = "=1.3.3", optional = true }
 bincode = { package = "bincode", version = "=2.0.0-rc.3", optional = true }
 bitcode = { version = "=0.6.3", optional = true }
 borsh = { version = "=1.5.1", features = ["derive"], optional = true }
+serde-brief = { version = "=0.1.0", features = [
```

Comment:
Move below `serde_bare` to keep this list in alphabetical order

Reply:
Whoops, renaming the crate made order requirements change. Adjusted it.